### ISSUE TITLE
Is it possible to pass context to ngxTippy template?

### ISSUE THREAD

**Issue by VirtusB** [farengeyt451/ngx-tippy-wrapper#43]
The way it's done with ngx-bootstrap is by passing [popoverContext] like so

```
<ng-template #tooltipTmpl let-view="view">
    <div class="tooltip-section">
        <div class="section-title">
            {{ view.dateRangeText }}
        </div>
        <div class="section-text">
            {{ view.data }}
        </div>
    </div>
</ng-template>
<div
   [popoverContext]="{ view: view }"
   [popover]="tooltipTmpl"
   triggers="mouseenter:mouseleave"
   placement="top"
>
  <div class="sub-title">
    {{ view.title }}
  </div>
</div>
```

I've looked through the documentation and source code, but I can't seem to find the way it's done with ngxTippy.

**Comment by farengeyt451** [comment]
Hey @VirtusB  it's not possible

**Comment by VirtusB** [comment]
> Hey @VirtusB it's not possible

I could try to submit a PR. 

In rough steps, how would I go about this? I haven't made a PR to OSS before, or a change to a package for Angular before.

Would I just clone the repo and run "ng serve" to test my changes?

Thanks

**Comment by onaluf** [comment]
I've added a pull request that implements this. It used to work by just setting the content of "name" as an object instead of a string and using it to pass a context to the template. This was probably a sort of a hack but it was even alluded to in the documentation.

That said, this was broken at some point by two things: proper typing of "name" forced it to be a string and a bug prevented name to be passed when using an `ng-template` tooltips.

The PR I've created fixes the bug and adds another input to handle the context separately from the name. Do you think there is any chance of having it reviewed? I would be very happy to fix anything you find objectionable !

**Comment by k290** [comment]
This would be a great feature to have. Hoping the PR makes its way into the lib.

**Comment by farengeyt451** [comment]
Resolved in v6.2.0
@onaluf Appreciated for contribution 😃
